infrastructure/terraform.tfvars.tmpl

```diff
@@ -16,7 +16,7 @@ stac_url="${VEDA_STAC_URL}"
 vector_secret_name="${VECTOR_SECRET_NAME}"
 vector_security_group="${VECTOR_SECURITY_GROUP}"
 vector_vpc="${VECTOR_VPC:-null}"
-workflow_root_path="${WORKFLOW_ROOT_PATH}"
+workflow_root_path="${WORKFLOW_ROOT_PATH:-/api/workflows}"
```

Comment:
This line ensures that the Terraform variable workflow_root_path will default to "/api/workflows" if WORKFLOW_ROOT_PATH is not defined in AWS Secret Manager. The only drawback is that the default value for workflow_root_path defined in variables.tf will always be ignored.
Probably a better solution is to change the name of the variable defined in AWS Secret Manager to TF_VAR_workflow_root_path and remove the line workflow_root_path="${WORKFLOW_ROOT_PATH:-/api/workflows}" from infrastructure/terraform.tfvars.tmpl. This way, Terraform will use the default value in variables.tf if TF_VAR_workflow_root_path is not defined in AWS Secret Manager.